aten/src/ATen/native/ReflectionPad.cpp

```diff
@@ -605,25 +580,56 @@ TORCH_IMPL_FUNC(reflection_pad1d_out_cpu)
   reflection_pad1d_out_template(output, input, padding);
 }
 
-Tensor& reflection_pad1d_backward_out_cpu(const Tensor& grad_output,
+TORCH_IMPL_FUNC(reflection_pad1d_backward_out_cpu)(const Tensor& grad_output_,
     const Tensor& input,
     IntArrayRef padding,
-    Tensor& grad_input) {
+    const Tensor& grad_input) {
   grad_input.resize_as_(input);
```

Comment:
IMPL functions shouldn't resize their inputs, they can assume on input that everything is correctly sized.